Formula/postgresql@14.rb

```diff
@@ -162,7 +162,7 @@ def caveats
   end
 
   service do
-    run [opt_bin/"postgres", "-D", f.postgresql_datadir]
+    run [opt_bin/"postgres", "-D", f.var/f.name]
```

Comment:
I don't think we want to change `f.postgresql_datadir` here.

Instead, I think we want to tell users to fix the `plist` after they've done the `mv` in the `caveats`.

Otherwise, you get a `brew services` command that doesn't work for users who didn't follow the `caveats`.

Reply:
It makes sense!

Reply:
Added to caveats!